### PR TITLE
Added `--json` flag

### PR DESCRIPTION
**Pull request by jenslind**
https://github.com/sindresorhus/speed-test/issues/14

### CODE REVIEW

**Comment by jenslind** [comment]
@sindresorhus any problem with this? :shipit:

File: cli.js [hunk 1]
```diff
@@ -41,21 +48,23 @@ function render() {
...
 
-setInterval(render, 50);
+if (!cli.flags.json) {
+	setInterval(render, 50);
+}
 
 st.once('testserver', function (server) {
 	state = 'download';
-	stats.ping = chalk.cyan(Math.round(server.bestPing) + chalk.dim(' ms'));
+	stats.ping = (cli.flags.json) ? Math.round(server.bestPing) : chalk.cyan(Math.round(server.bestPing) + chalk.dim(' ms'));
```
Comment (sindresorhus) [inline cli.js]: We're now rounding twice. Probably worth putting it in a variable and use it. Same applies for the below events.

File: cli.js [hunk 2]
```diff
@@ -31,6 +33,11 @@ function getSpinner(x) {
 }
 
 function render() {
+	if (cli.flags.json) {
+		logUpdate(JSON.stringify(stats));
```
Comment (sindresorhus) [inline cli.js]: Use `console.log`

File: cli.js [hunk 3]
```diff
@@ -11,7 +11,9 @@ var elegantSpinner = require('elegant-spinner');
 var cli = meow({
 	help: [
 		'Usage',
-		'  $ speed-test'
+		'  $ speed-test',
+		'Flags',
```
Comment (sindresorhus) [inline cli.js]: `Flags` => `Options`

Comment (sindresorhus) [inline cli.js]: Also need to update the readme with the new CLI help output.

File: cli.js [hunk 4]
```diff
@@ -11,7 +11,9 @@ var elegantSpinner = require('elegant-spinner');
...
+		'  --json'
```
Comment (sindresorhus) [inline cli.js]: Could use a short description

**Comment by sindresorhus** [comment]
Sorry about the delay. I've been afk for some days. Looks pretty good, thanks :) Some minor comments inline.

**Comment by jenslind** [comment]
No worries :) Updated the PR.

**Pull request closed without merging**